pageserver/src/metrics.rs

```diff
@@ -1304,10 +1342,42 @@ use std::time::{Duration, Instant};
 use crate::context::{PageContentKind, RequestContext};
 use crate::task_mgr::TaskKind;
 
+struct PerTimelineRemotePhysicalSizeGauge {
```

Comment:
```suggestion
/// Maintain a per timeline gauge in addition to the global gauge.
struct PerTimelineRemotePhysicalSizeGauge {
```

Reply:
Not really important, but the global aspect is not in the name... Well, perhaps you and me are the only ones looking at these :) Feel free not to do but just resolve.